README.md

```diff
@@ -18,7 +18,7 @@ npm install --save hoodie-server-store
 ## Example
 
 ```js
-var Hapi = require('hapi')
+var hapi = require('hapi')
```

Comment:
Yeah but something like a Class Bundle? `new hapi.Server` looks somewhat weird to me, and as `Hapi.Server` is the convention for hapi developers, I’d prefer to stick with that. Would you mind?